.mailmap

```diff
@@ -0,0 +1,110 @@
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@Grumpy.dyn.lifesci.dundee.ac.uk>
+Douglas Creager <douglas.creager@comlab.ox.ac.uk> <dcreager@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Douglas Russell <douglas.russell@bioch.ox.ac.uk> dpwrussell <douglas.russell@bioch.ox.ac.uk>
+Douglas Russell <douglas.russell@bioch.ox.ac.uk> <douglas_russell@hms.harvard.edu>
+Emil Rozbicki <emil@glencoesoftware.com> <emilroz@gmail.com>
+Harry Hochheiser <harryh@pitt.edu> <hsh@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Helen Flynn <h.flynn@dundee.ac.uk> <omehelen@gmail.com>
+Jean-Marie Burel <j.burel@dundee.ac.uk> jburel <j.burel@dundee.ac.uk>
+Jean-Marie Burel <j.burel@dundee.ac.uk> jean-marie burel <j.burel@dundee.ac.uk>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@05709c45-44f0-0310-885b-81a1db45b4a6>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@users.noreply.github.com>
+Jeff Mellen <jeffm@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jesse Corrington <jesse@glencoesoftware.com> <jesse.corrington@gmail.com>
+Josh Moore <josh@openmicroscopy.org> <j.a.moore@dundee.ac.uk>
+Josh Moore <josh@openmicroscopy.org> <josh@glencoesoftware.com>
+Josh Moore <josh@openmicroscopy.org> <josh@example.com>
+Josh Moore <josh@openmicroscopy.org> <josh.moore@gmx.de>
+Josh Moore <josh@openmicroscopy.org> <josh@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Josh Moore <josh@openmicroscopy.org> <jmoore@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Joshua Ballanco <jballanc@glencoesoftware.com> Josh Ballanco <jballanc@glencoesoftware.com>
+Joshua Ballanco <jballanc@glencoesoftware.com> <jballanc@gmail.com>
+Kenny Gillen <k.h.gillen@dundee.ac.uk> Kenneth Gillen <k.h.gillen@dundee.ac.uk>
+Kouichi Nakamura <kouichi.c.nakamura@gmail.com> Kouichi C. Nakamura <kouichi.c.nakamura@gmail.com>
+Kouichi Nakamura <kouichi.c.nakamura@gmail.com> KOUICHI NAKAMURA <kouichi.c.nakamura@gmail.com>
+Liza Unson <lunson@glencoesoftware.com> Liza <lunson@glencoesoftware.com>
+Luca Lianas <luca.lianas@crs4.it> <lianas@crs4.it>
+Mark Carroll <m.t.b.carroll@dundee.ac.uk> <M.T.B.Carroll@Dundee.ac.uk>
+Mark Carroll <m.t.b.carroll@dundee.ac.uk> <mtbc@ls28101>
+Muhanad Zahra <muhanad@glencoesoftware.com> <Muhanad.Zahra@outlook.com>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa.linkert@gmail.com>
+Melissa Linkert <melissa@glencoesoftware.com> <mlinkert-x@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Melissa Linkert <melissa@glencoesoftware.com> <melissa@05709c45-44f0-0310-885b-81a1db45b4a6>
+Nikolaus Ehrenfeuchter <nikolaus.ehrenfeuchter@unibas.ch>
```

Comment:
should probably be 
``Nikolaus Ehrenfeuchter <nikolaus.ehrenfeuchter@unibas.ch> Niko Ehrenfeuchter <nikolaus.ehrenfeuchter@unibas.ch>``